.travis.yml

```diff
@@ -34,12 +54,13 @@ before_script:
   - bin/console doctrine:database:create --env=test
 
 script:
-  - bin/phpcs --standard=app/phpcs.xml src
-  - bin/phpunit --coverage-clover build/logs/clover.xml
-  - bin/console doctrine:schema:validate --env=dev
+  - if [ "$VALIDATE_SCHEMA" = true ]; then (bin/console doctrine:schema:validate --env=dev); fi
+  - if [ "$CHECK_CODING_STANDARDS" = true ]; then (bin/phpcs --standard=app/phpcs.xml src); fi
+  - if [ "$TEST_PHPUNIT_CONTROLLERS" = true ]; then (bin/phpunit -c phpunit.xml.dist --group controllers --coverage-clover build/logs/clover.xml); fi
+  - if [ "$TEST_PHPUNIT_OTHERS" = true ]; then (bin/phpunit -c phpunit.xml.dist --exclude-group mesh_data_import,controllers --coverage-clover build/logs/clover.xml); fi
 
 after_script:
-  - php bin/coveralls #send test coverage informatio to coveralls
+  - if [ "$TEST_PHPUNIT_CONTROLLERS" = true ] || [ "$TEST_PHPUNIT_OTHERS" = true ]; then (php bin/coveralls); fi #send test coverage information to coveralls
```

Comment:
Probably just pull this out since the tests are split the data we send will be wrong.  We will have to keep up with coverage on our own.